__tests__/WebAnnotation.test.js

```diff
@@ -36,21 +35,29 @@ describe('WebAnnotation', () => {
             value: 'svg',
           },
         ],
+        source: 'canvasId',
       });
     });
     it('with svg only', () => {
       subject = createSubject({ xywh: null });
       expect(subject.target()).toEqual({
-        id: 'canvasId',
         selector: {
           type: 'SvgSelector',
           value: 'svg',
         },
+        source: 'canvasId',
       });
     });
     it('with xywh only', () => {
       subject = createSubject({ svg: null });
-      expect(subject.target()).toBe('canvasId#xywh=xywh');
+      /* expect(subject.target()).toBe('canvasId#xywh=xywh'); */
```

Comment:
Should we remove this comment?

Reply:
Sure, I had left it as a reminder to discuss that WebAnnotation now creates a SpecificResource with a FragmentSelector in all cases. The earlier code would create just a fragment URI if there was only a `xywh` selector (which does not happen anyway).